proxy/utils/ip.ts

```diff
@@ -0,0 +1,28 @@
+import { isIPv4, isIPv6 } from 'net'
+
+export function stripPort(ip: string) {
```

Comment:
This function is a little hard to follow. We already got `isIPv4` and `isIPv6` functions, can't we simplify the implementation using these methods, like below?

```
if (isIPv4(ip)) {
 const colonIndex = ip.indexOf(':')
 if (colonIndex === -1) {
  return ip
 }
 
 return ip.substring(0, colonIndex)
} else if (isIPv6(ip)) {
 const openingBracketIndex = ip.indexOf('[')
 const closingBracketIndex = ip.indexOf(']')
 if (openingBracketIndex === 0 && closingBracketIndex > -1) {
          return ip.substring(1, closingBracketIndex)
  }
  
  return ip
} else {
 return ip // or return empty string maybe? ip is clearly malformed
}
```

Reply:
Understood 😢